recipe/meta.yaml

```diff
@@ -103,7 +103,7 @@ requirements:
     - {{native}}libpng                   # [win]
     - pango                              # [not win]
     - {{native}}icu                      # [win]
-    - icu                                # [not win]
+    - icu 58.2                           # [not win]
```

Comment:
I don't think this is a good idea. We need to follow the overall pinning of conda-forge here.

Reply:
Yea, I'm trying to find ways around that. Texlive-core 20180414 doesn't work with the latest poppler as far as I can tell (I tried). The current recipe has a patch to work with poppler 0.64/0.65 and I found a separate patch for to work with poppler 0.66, but the latest version (0.80) fails the texlive config. Poppler in the 0.60's has a dependency on cairo 1.14 and cairo 1.14 depends on ICU 58.2.

The current r-base recipe has icu 64.2, but I'm curious how that works since there is no texlive version that *should* work with those levels. I must be missing something..

Reply:
For an experiement in the CI, I removed the pin. Now we get ICU mismatches:

```
../../lib/libR.so: undefined reference to `u_versionToString_58'
..
```

Reply:
@mingwandroid What do you think? 

We could try refreshing texlive up to the 2019 level, and maybe that works with the newer poppler/cairo/icu versions. I've not tried that.

Reply:
That issue is because R is picking up `icu` from the build environment.

Reply:
Well, it is building R at the time, right?

Reply:
Yes. Try adding `icu` to build as well. (I know it's a hack, but let's see if that works)

Reply:
ah ok

Reply:
I thought it was working for a minute, but still failed the same..

Reply:
@isuruf Any other ideas? It looks like the oldest version of poppler that depends on the *newer* cairo (and therefor the newer ICU) is poppler 0.78. I didn't try that version specifically with a Texlive package rebuild, but 0.80 was definitely too new.

The older TexLive build didn't have a poppler dependency, but it still required cairo 1.14, which wants ICU 58, so I'm still curious how r-base works currently with the build req on ICU 64..